### DISCUSSION TITLE
баг термукс бокса

### DISCUSSION THREAD

**Discussion by 20narayan** [olegos2/termux-box#47] [General]
Здравствуйте Хуго и Олег,сразу напишу тестируемое устройство realme gt neo 3t с 870снап и 8/128 памяти,запускаю на только что скачанном и установленном термукс-бокс.
Суть в том что после второго или третьего выхода из бокса перестает загружаться ТФМ,и даже из меню старт он не работает. Я записал видео и одновременно снял лог.

**Comment by 20narayan** [comment]
Dynarec will play moderatly safe with x64 flags
Dynarec for ARM64, with extension: ASIMD AES CRC32 PMULL ATOMICS PageSize:4096 Running on Cortex-A77 with 8 Cores
Warning, unsupported BOX64_DYNAREC_JVM=0 for [projectzomboid64] in /etc/box64.box64rc
Params database has 24 entries
BOX64: Wine64 detected, WINEPRELOADRESERVE=""
Using default BOX64_LD_LIBRARY_PATH: ./:lib/:lib64/:x86_64/:bin64/:libs64/
Using default BOX64_PATH: ./:bin/
Counted 37 Env var
Looking for wine64
argv[1]="explorer"
argv[2]="/desktop=shell,960x540"
argv[3]="/opt/TFM.exe"
Rename process to "wine64"
Using native(wrapped) libdl.so.2
Using native(wrapped) libc.so.6
Using native(wrapped) ld-linux-x86-64.so.2
Using native(wrapped) libpthread.so.0
Using native(wrapped) libutil.so.1
Using native(wrapped) librt.so.1
Using emulated /opt/wine/bin/../lib/wine/x86_64-unix/ntdll.so
Using emulated /lib/x86_64-linux-gnu/libgcc_s.so.1
Dynarec will play moderatly safe with x64 flags
Dynarec for ARM64, with extension: ASIMD AES CRC32 PMULL ATOMICS PageSize:4096 Running on Cortex-A77 with 8 Cores
Warning, unsupported BOX64_DYNAREC_JVM=0 for [projectzomboid64] in /etc/box64.box64rc
Params database has 24 entries
BOX64: Wine64 detected, WINEPRELOADRESERVE=""
Using default BOX64_LD_LIBRARY_PATH: ./:lib/:lib64/:x86_64/:bin64/:libs64/
Using default BOX64_PATH: ./:bin/
Counted 39 Env var
Looking for /opt/wine/bin/wine64
argv[1]="explorer"
argv[2]="/desktop=shell,960x540"
argv[3]="/opt/TFM.exe"
Rename process to "wine64"
Using native(wrapped) libdl.so.2
Using native(wrapped) libc.so.6
Using native(wrapped) ld-linux-x86-64.so.2
Using native(wrapped) libpthread.so.0
Using native(wrapped) libutil.so.1
Using native(wrapped) librt.so.1
Using emulated /opt/wine/bin/../lib/wine/x86_64-unix/ntdll.so
Using emulated /lib/x86_64-linux-gnu/libgcc_s.so.1
Dynarec will play moderatly safe with x64 flags
Dynarec for ARM64, with extension: ASIMD AES CRC32 PMULL ATOMICS PageSize:4096 Running on Cortex-A77 with 8 Cores
Warning, unsupported BOX64_DYNAREC_JVM=0 for [projectzomboid64] in /etc/box64.box64rc
Params database has 24 entries
Using default BOX64_LD_LIBRARY_PATH: ./:lib/:lib64/:x86_64/:bin64/:libs64/
Using default BOX64_PATH: ./:bin/
Counted 39 Env var
Looking for /opt/wine/bin/wineserver
Rename process to "wineserver"
Using native(wrapped) libdl.so.2
Using native(wrapped) libc.so.6
Using native(wrapped) ld-linux-x86-64.so.2
Using native(wrapped) libpthread.so.0
Using native(wrapped) libutil.so.1
Using native(wrapped) librt.so.1
Using emulated /opt/wine/lib/wine/x86_64-unix/libwine.so.1
0034:err:seh:install_bpf Native libs are being loaded in low addresses, sc_seccomp 0x10204b050, syscall 0x50560, not installing seccomp.
0034:err:seh:install_bpf The known reasons are /proc/sys/vm/legacy_va_layout set to 1 or 'ulimit -s' being 'unlimited'.
0034:fixme:winediag:LdrInitializeThunk wine-staging 8.2 is a testing version containing experimental patches.
0034:fixme:winediag:LdrInitializeThunk Please mention your exact version when filing bug reports on winehq.org.
0034:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (2).
0044:err:seh:install_bpf Native libs are being loaded in low addresses, sc_seccomp 0x10204b050, syscall 0x50560, not installing seccomp.
0044:err:seh:install_bpf The known reasons are /proc/sys/vm/legacy_va_layout set to 1 or 'ulimit -s' being 'unlimited'.
0050:err:seh:install_bpf Native libs are being loaded in low addresses, sc_seccomp 0x10204b050, syscall 0x50560, not installing seccomp.
0050:err:seh:install_bpf The known reasons are /proc/sys/vm/legacy_va_layout set to 1 or 'ulimit -s' being 'unlimited'.
0074:err:seh:install_bpf Native libs are being loaded in low addresses, sc_seccomp 0x10204b050, syscall 0x50560, not installing seccomp.
0074:err:seh:install_bpf The known reasons are /proc/sys/vm/legacy_va_layout set to 1 or 'ulimit -s' being 'unlimited'.
0088:err:wineusb:usb_init Failed to initialize libusb: Other error
0084:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
0094:err:seh:install_bpf Native libs are being loaded in low addresses, sc_seccomp 0x10204b050, syscall 0x50560, not installing seccomp.
0094:err:seh:install_bpf The known reasons are /proc/sys/vm/legacy_va_layout set to 1 or 'ulimit -s' being 'unlimited'.
0084:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
0084:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
0084:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
00b4:err:seh:install_bpf Native libs are being loaded in low addresses, sc_seccomp 0x10204b050, syscall 0x50560, not installing seccomp.
00b4:err:seh:install_bpf The known reasons are /proc/sys/vm/legacy_va_layout set to 1 or 'ulimit -s' being 'unlimited'.
0024:err:seh:install_bpf Native libs are being loaded in low addresses, sc_seccomp 0x10204b050, syscall 0x50560, not installing seccomp.
0024:err:seh:install_bpf The known reasons are /proc/sys/vm/legacy_va_layout set to 1 or 'ulimit -s' being 'unlimited'.
Using emulated /opt/wine/lib/wine/x86_64-unix/shell32.dll.so
Using emulated /opt/wine/lib/wine/x86_64-unix/win32u.so
Using native(wrapped) libm.so.6
Using native(wrapped) libfreetype.so.6
Using native(wrapped) libfontconfig.so.1
Dynarec will play moderatly safe with x64 flags
Dynarec for ARM64, with extension: ASIMD AES CRC32 PMULL ATOMICS PageSize:4096 Running on Cortex-A77 with 8 Cores
Warning, unsupported BOX64_DYNAREC_JVM=0 for [projectzomboid64] in /etc/box64.box64rc
Params database has 24 entries
BOX64: Wine64 detected, WINEPRELOADRESERVE="000400000-000424000"
Using default BOX64_LD_LIBRARY_PATH: ./:lib/:lib64/:x86_64/:bin64/:libs64/
Using default BOX64_PATH: ./:bin/
Counted 41 Env var
Looking for /opt/wine/bin/wine64
argv[1]="C:\windows\system32\explorer.exe"
argv[2]="/desktop=shell,960x540"
argv[3]="/opt/TFM.exe"
Rename process to "wine64"
Using native(wrapped) libdl.so.2
Using native(wrapped) libc.so.6
Using native(wrapped) ld-linux-x86-64.so.2
Using native(wrapped) libpthread.so.0
Using native(wrapped) libutil.so.1
Using native(wrapped) librt.so.1
Using emulated /opt/wine/bin/../lib/wine/x86_64-unix/ntdll.so
Using emulated /lib/x86_64-linux-gnu/libgcc_s.so.1
Using emulated /opt/wine/lib/wine/x86_64-unix/libwine.so.1
00cc:err:seh:install_bpf Native libs are being loaded in low addresses, sc_seccomp 0x10204b050, syscall 0x50560, not installing seccomp.
00cc:err:seh:install_bpf The known reasons are /proc/sys/vm/legacy_va_layout set to 1 or 'ulimit -s' being 'unlimited'.
Using emulated /opt/wine/lib/wine/x86_64-unix/win32u.so
Using native(wrapped) libm.so.6
Using native(wrapped) libfreetype.so.6
Using native(wrapped) libfontconfig.so.1
Using emulated /opt/wine/lib/wine/x86_64-unix/winex11.so
Using native(wrapped) libXext.so.6
Using native(wrapped) libX11.so.6
Using native(wrapped) libxcb.so.1
Using native(wrapped) libXau.so.6
Using native(wrapped) libXdmcp.so.6
Using native(wrapped) libXinerama.so.1
Using native(wrapped) libXxf86vm.so.1
Using native(wrapped) libXrender.so.1
Using native(wrapped) libXrandr.so.2
Using native(wrapped) libXcomposite.so.1
Using native(wrapped) libXi.so.6
Using native(wrapped) libXcursor.so.1
Using native(wrapped) libXfixes.so.3
Using emulated /opt/wine/lib/wine/x86_64-unix/shell32.dll.so
Dynarec will not try to make big blocks
Dynarec will play moderatly safe with x86 flags
Dynarec for ARMv8, with extension: HALF FAST_MULT EDSP NEON VFPv4 IDIVA PageSize:4096 Running on AArch64 Processor rev 0 (aarch64)
 with 8 Cores
Params database has 22 entries
BOX86: Wine preloader detected, loading "/opt/wine/bin/wine" directly
BOX86: Wine detected, WINEPRELOADRESERVE="000400000-000421000"
Using default BOX86_LD_LIBRARY_PATH: ./:lib/:lib32/:x86/:i686/
Using default BOX86_PATH: ./:bin/
Counted 41 Env var
Looking for /opt/wine/bin/wine
argv[1]="/opt/TFM.exe"
Rename process to "wine"
Using native(wrapped) libdl.so.2
Using native(wrapped) libpthread.so.0
Using native(wrapped) libc.so.6
Using native(wrapped) ld-linux.so.2
Using native(wrapped) librt.so.1
Using emulated /opt/wine/bin/../lib/wine/i386-unix/ntdll.so
Using native(wrapped) libm.so.6
Using emulated /opt/wine/lib/wine/i386-unix/libwine.so.1
Using emulated /opt/wine/lib/wine/i386-unix/shell32.dll.so
00fc:err:seh:install_bpf Native libs are being loaded in low addresses, sc_seccomp 0x10204b050, syscall 0x50560, not installing seccomp.
00fc:err:seh:install_bpf The known reasons are /proc/sys/vm/legacy_va_layout set to 1 or 'ulimit -s' being 'unlimited'.
Using emulated /opt/wine/lib/wine/i386-unix/win32u.so
Using native(wrapped) libfreetype.so.6
Using native(wrapped) libfontconfig.so.1
Using emulated /opt/wine/lib/wine/i386-unix/winex11.so
Using native(wrapped) libXext.so.6
Using native(wrapped) libX11.so.6
Using native(wrapped) libxcb.so.1
Using native(wrapped) libXau.so.6
Using native(wrapped) libXdmcp.so.6
Using native(wrapped) libXinerama.so.1
Using native(wrapped) libXxf86vm.so.1
Using native(wrapped) libXrender.so.1
Using native(wrapped) libXrandr.so.2
Using native(wrapped) libXcomposite.so.1
Using native(wrapped) libXi.so.6
Using native(wrapped) libXcursor.so.1
Using emulated /opt/wine/lib/wine/i386-unix/bcrypt.so
Using native(wrapped) libgnutls.so.30
Using emulated /opt/wine/lib/wine/i386-unix/crypt32.so
wine: Unhandled page fault on read access to 00000073 at address 007333F4 (thread 0148), starting debugger...
Dynarec will not try to make big blocks
Dynarec will play moderatly safe with x86 flags
Dynarec for ARMv8, with extension: HALF FAST_MULT EDSP NEON VFPv4 IDIVA PageSize:4096 Running on AArch64 Processor rev 0 (aarch64)
 with 8 Cores
Params database has 22 entries
BOX86: Wine detected, WINEPRELOADRESERVE="000400000-000467000"
winedbg detected, not launching it!
0148:err:seh:start_debugger Couldn't start debugger L"winedbg --auto 232 160" (1359)
Read the Wine Developers Guide on how to set up winedbg or another debugger
00cc:fixme:imm:ImeSetActiveContext (0000000000010026, 0): stub
00cc:fixme:imm:ImmReleaseContext (0000000000010020, 0000000000010026): stub
Using emulated /opt/wine/lib/wine/x86_64-unix/bcrypt.so
Using native(wrapped) libgnutls.so.30
Using emulated /opt/wine/lib/wine/x86_64-unix/winspool.so
Using emulated /opt/wine/lib/wine/x86_64-unix/ws2_32.so
00cc:fixme:heap:RtlSetHeapInformation handle 00000000022D0000, info_class 0, info 000000000062CBE0, size 4 stub!
Using emulated /opt/wine/lib/wine/x86_64-unix/bcrypt.so
Using native(wrapped) libgnutls.so.30
Using emulated /opt/wine/lib/wine/x86_64-unix/winspool.so
Using emulated /opt/wine/lib/wine/x86_64-unix/ws2_32.so
00cc:fixme:heap:RtlSetHeapInformation handle 00000000026E0000, info_class 0, info 000000000062C3F0, size 4 stub!
fixme:shell:ISF_ControlPanel_fnGetDisplayNameOf retrieve display name from control panel app
fixme:shell:ISF_ControlPanel_fnGetDisplayNameOf retrieve display name from control panel app
fixme:shell:ISF_ControlPanel_fnGetDisplayNameOf retrieve display name from control panel app
fixme:exec:SHELL_execute flags ignored: 0x00000004
fixme:exec:SHELL_execute flags ignored: 0x00004000
wine: Unhandled page fault on read access to 42E80073 at address 00733400 (thread 017c), starting debugger...
017c:err:seh:start_debugger Couldn't start debugger L"winedbg --auto 364 156" (1359)
Read the Wine Developers Guide on how to set up winedbg or another debugger

**Comment by 20narayan** [comment]
Вот видео этого лога https://youtu.be/KUUg-dEF2rk

**Comment by Romaxaxa22** [comment]
Здравствуйте. Обновился до 14 android, термукс не устанавливается, как исправить?

**Comment by MrMinyUkho** [comment]
Привет, во время первого запуска словил баг wget, при попытке загрузить вайн он не может создать ubuntu-fs/opt, пишет что нет файла или директории, фиксится созданием этих папок и загрузка идёт
